src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -106,7 +106,7 @@ public void GetAccessControl_FileInfo_AccessControlSections_ReturnsValidObject()
         [Fact]
         public void GetAccessControl_Filestream_InvalidArguments()
         {
-            Assert.Throws<NullReferenceException>(() => FileSystemAclExtensions.GetAccessControl((FileStream)null));
+            Assert.Throws<ArgumentNullException>("fileStream", () => FileSystemAclExtensions.GetAccessControl((FileStream)null));
```

Comment:
The CI hit failures because the NetFX version of `FileSystemAclExtensions` needs to get the nullchecks added as well:

https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.net46.cs#L33-L36

Can you please add those too, @eanova?

FYI here's the callstack of the unit test that failed:
https://helix.dot.net/api/2019-06-17/jobs/05c5ed6b-44d3-42c6-9de1-4a321c65d99d/workitems/System.IO.FileSystem.AccessControl.Tests/console

```
 System.IO.FileSystemAclExtensionsTests.SetAccessControl_FileStream_FileSecurity_InvalidArguments [FAIL]
      Assert.Throws() Failure
      Expected: typeof(System.ArgumentNullException)
      Actual:   typeof(System.NullReferenceException): Object reference not set to an instance of an object.
      ---- System.NullReferenceException : Object reference not set to an instance of an object.
      Stack Trace:
        /_/src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs(178,0): at System.IO.FileSystemAclExtensionsTests.<>c.<SetAccessControl_FileStream_FileSecurity_InvalidArguments>b__15_0()

...

    System.IO.FileSystemAclExtensionsTests.GetAccessControl_Filestream_InvalidArguments [FAIL]
      Assert.Throws() Failure
      Expected: typeof(System.ArgumentNullException)
      Actual:   typeof(System.NullReferenceException): Object reference not set to an instance of an object.
      ---- System.NullReferenceException : Object reference not set to an instance of an object.
      Stack Trace:
        /_/src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs(109,0): at System.IO.FileSystemAclExtensionsTests.<>c.<GetAccessControl_Filestream_InvalidArguments>b__9_0()
```

Reply:
Before I submit the changes do you mind if we go back to Allman bracing for the null checks?

Reply:
The issue with NetFX implementation is fixed now.  Unfortunately one of the builds failed during startup.